### PR TITLE
Modify project design

### PR DESCRIPTION
**Pull request by Tshepo45**
Change project title
Fix .css typos
Modify README.md file

### CODE REVIEW

**Review by NduatiKagiri — COMMENTED** [review]
Hi @Tshepo45 ,

# INVAILD REVIEW REQUEST ❌ ❌  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have any code. Please make sure your code is available in your Pull Request. And also, this project milestone is a continuation of the previous milestone, so kindly make sure you that the code for the previous milestone is available in this repository's main branch.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -39,8 +39,7 @@ <h1>I'm Tshepo <br> Glad to see <br> you!</h1>
 </div>
 </section>
 
-  
- <!--work section-->
+<!--work section-->
  <section id="work-section">
   <div class="container1">
     <figure class="snap-shoot-one">
```
Comment (NahnahAJ) [inline index.html:45]: - [ ] Your project is incomplete. I don't know if you already merged before submitting for review but there is nothing in these files to review. These selected lines are the only new changes introduced in your index.html file. 
- [ ] Aside from that, Your project still has a long way to go. I would suggest you start the project all over and resubmit for review.
- [ ] Kindly try to stick to the Figma Design Template you selected :100:
-This is how your project looks currently. 
https://www.loom.com/share/fccebe01e8294c88bc5b5332c97b6e76
- [ ] Please  follow the project requirements and the design guidelines and redo them before submitting for review.

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
Hi @Tshepo45 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Good README file :100: 
- Good commit messages :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -66,6 +67,35 @@ <h1>Tonic</h1>
...
   
+=======
+ 
+<div id="container">
+    <div class="container1">
+        <div class="Snap-Shoot1">
+            <img src="image/Snapshoot Portfolio6.png" alt="Snap-Shoot1">
+        </div>
+        <div class="Project-Text">
+            <h1>Tonic</h1>
```
Comment (IngDavidTM) [inline index.html:78]: - [ ] Good job identifying the titles and subtitles :+1:, but the title (e.g., "My Recent Works") doesn't have to use `<h1>` tag. You could use `<h2>` tag. This is essentially due to SEO reasons which many believe it is better to have only one `<h1>` tag on a webpage.

File: index.html [hunk 3]
```diff
@@ -66,6 +67,35 @@ <h1>Tonic</h1>
...
+    <ul class="Pro-P">
+        <li> <img class="Canopy" src="image/Text 4.png" alt="CANOPY"/> </li>
+
+        <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
+        
+        <li> <img class="Back-End" src="image/Back-end.png" alt="Back End Dev" /> </li>
+
+        <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
+        
+        <li> <img class="Year" src="image/2015.png" alt="2015" /> </li>
```
Comment (IngDavidTM) [inline index.html:89]: - [ ] While you did a great effort :+1: , as a project requirement each `<li>` element shouldn't be an image

File: styles.css [hunk 4]
```diff
@@ -8,6 +8,7 @@ body {
   background-size: cover;
   height: 100vh;
   width: 100%;
+  overflow: hidden;
```
Comment (IngDavidTM) [inline styles.css:11]: - [IMPORTANT] I would suggest removing this line because it is not allowing to show the cards of your project :+1:

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @Tshepo45 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [ ] Good job creating all the required elements inside your project :+1:, but I noticed that your HTML Structure is not using the right HTML semantics. The section element is a generic standalone section of the document, so use it to group elements that are related together. There are two main sections in the project that you should create, the works section and the about myself section, so please create them :pray: . To know more about `<section>` tag [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

- [ ] I've noticed that some of your tags like `<div>`, `<body>` and `<html>` tags are not closed so please check your document to close each of them because it is semantically correct :+1:. To know more about it you can check [this](https://simpledev.io/html/html-opening-and-closing-tags/)

- [ ] Good job creating each card, but I've noticed that the first card doesn't have the button tag, and also it is not aligned like the other cards, so please check the code to see what is happening :pray: 

- [ ] While your project implements some changes in the work section :+1:, like the last reviewer said, there is some lines that you didn't implement here so we can not review the other requirements like 'about me' section. so I would suggest you start the project all over and resubmit for review, al least the other section.

- [ ] Good job styling your project but I've noticed that your main page could move from right to left as you can see [here](https://www.loom.com/share/b3d3dfa796a14008b0ea5198ae68b2e5).


- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -123,15 +119,13 @@ <h2>Multi-Post<br>Stories</h2>
...
                 <h3>Tonic</h3>
                 <ul class="Pro-P">
-                    <li> <img class="Canopy" src="image/Text 4.png" alt="CANOPY"/> </li>
-            
-                    <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
-                    
-                    <li> <img class="Back-End" src="image/Back-end.png" alt="Back End Dev" /> </li>
-            
-                    <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
+                  <li> <p class="Canopy">Canopy</p></li>
```
Comment (MaganaAsiati) [inline index.html:122]: - [ ] [ OPTIONAL] KindlyI suggest you only use the` <p> ` tag for a paragraph it for a sematic reasons:+1:  Here above you can just use the simple `<div> or <h4>`

File: index.html [hunk 6]
```diff
@@ -123,15 +119,13 @@ <h2>Multi-Post<br>Stories</h2>
...
+                  <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/></li>
+                  
+                  <li> <p class="Back-End">Back-End Dev</p> </li>
```
Comment (MaganaAsiati) [inline index.html:125]: - [ ] [ OPTIONAL] KindlyI suggest you only use the` <p> ` tag for a paragraph it for a sematic reasons:+1:  Here above you can just use the simple `<div> or <h4>`

File: index.html [hunk 7]
```diff
@@ -123,15 +119,13 @@ <h2>Multi-Post<br>Stories</h2>
...
+                  <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
                     
-                    <li> <img class="Year" src="image/2015.png" alt="2015" /> </li>
+                  <li> <p class="Year">2015</p>
```
Comment (MaganaAsiati) [inline index.html:128]: - [ ] [ OPTIONAL] KindlyI suggest you only use the` <p> ` tag for a paragraph it for a sematic reasons:+1:  Here above you can just use the simple `<div> or <h4>`

File: index.html [hunk 8]
```diff
@@ -160,15 +154,13 @@ <h3>Tonic</h3>
             <div class="Project-Text">
               <h4>Multi-Post<br>Stories</h4>
                 <ul class="Pro-P">
-                    <li> <img class="Canopy" src="image/Text 4.png" alt="CANOPY"/> </li>
-            
-                    <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
+                  <li> <p class="Canopy">Canopy</p></li>
```
Comment (MaganaAsiati) [inline index.html:157]: - [ ] [ OPTIONAL] KindlyI suggest you only use the` <p> ` tag for a paragraph it for a sematic reasons:+1:  Here above you can just use the simple `<div> or <h4>`

File: index.html [hunk 9]
```diff
@@ -160,15 +154,13 @@ <h3>Tonic</h3>
...
+                  <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/></li>
+                  
+                  <li> <p class="Back-End">Back-End Dev</p> </li>
```
Comment (MaganaAsiati) [inline index.html:160]: - [ ] [ OPTIONAL] KindlyI suggest you only use the` <p> ` tag for a paragraph for a sematic reasons:+1:  Here above you can just use the simple `<div> or <h4>`

File: index.html [hunk 10]
```diff
@@ -160,15 +154,13 @@ <h3>Tonic</h3>
...
+                  <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
                     
-                    <li> <img class="Back-End" src="image/Back-end.png" alt="Back End Dev" /> </li>
-            
-                    <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
-                    
-                    <li> <img class="Year" src="image/2015.png" alt="2015" /> </li>
+                  <li> <p class="Year">2015</p>
```
Comment (MaganaAsiati) [inline index.html:163]: - [ ] [ OPTIONAL] KindlyI suggest you only use the` <p> ` tag for a paragraph it for a sematic reasons:+1:  Here above you can just use the simple `<div> or <h4>`

File: index.html [hunk 11]
```diff
@@ -86,15 +84,13 @@ <h1>Tonic</h1>
...
-            
-                    <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
+                  <li> <p class="Canopy">Canopy</p></li>
+                  <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/></li>
+                  
+                  <li> <p class="Back-End">Back-End Dev</p> </li>
+                  <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
                     
-                    <li> <img class="Year" src="image/2015.png" alt="2015" /> </li>
+                  <li> <p class="Year">2015</p>
```
Comment (MaganaAsiati) [inline index.html:93]: - [ ] [ OPTIONAL] KindlyI suggest you only use the` <p> ` tag for a paragraph it for a sematic reasons:+1:  Here above you can just use the simple `<div> or <h4>`

File: index.html [hunk 12]
```diff
@@ -86,15 +84,13 @@ <h1>Tonic</h1>
...
                 <h2>Multi-Post<br>Stories</h2>
                 <ul class="Pro-P">
-                    <li> <img class="Canopy" src="image/Text 4.png" alt="CANOPY"/> </li>
-            
-                    <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
-                    
-                    <li> <img class="Back-End" src="image/Back-end.png" alt="Back End Dev" /> </li>
-            
-                    <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
+                  <li> <p class="Canopy">Canopy</p></li>
```
Comment (MaganaAsiati) [inline index.html:87]: - [ ] [ OPTIONAL] KindlyI suggest you only use the` <p> ` tag for a paragraph for a sematic reasons:+1:  Here above you can just use the simple `<div> or <h4>`

File: index.html [hunk 13]
```diff
@@ -39,26 +39,24 @@ <h1>I'm Tshepo <br> Glad to see <br> you!</h1>
...
         
-        <li> <img class="Back-End" src="image/Back-end.png" alt="Back End Dev" /> </li>
-
+        <li> <p class="Back-End">Back-End Dev</p> </li>
         <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
-        
-        <li> <img class="Year" src="image/2015.png" alt="2015" /> </li>
-      </ul>
+          
+        <li> <p class="Year">2015</p>
```
Comment (MaganaAsiati) [inline index.html:58]: - [ ] [ OPTIONAL] KindlyI suggest you only use the` <p> ` tag for a paragraph it for a sematic reasons:+1:  Here above you can just use the simple `<div> or <h4>`

File: index.html [hunk 14]
```diff
@@ -39,26 +39,24 @@ <h1>I'm Tshepo <br> Glad to see <br> you!</h1>
...
     <ul class="Pro-P">
-        <li> <img class="Canopy" src="image/Text 4.png" alt="CANOPY"/> </li>
-
-        <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
+        <li> <p class="Canopy">Canopy</p></li>
+        <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/></li>
         
-        <li> <img class="Back-End" src="image/Back-end.png" alt="Back End Dev" /> </li>
-
+        <li> <p class="Back-End">Back-End Dev</p> </li>
```
Comment (MaganaAsiati) [inline index.html:55]: - [ ] [ OPTIONAL] KindlyI suggest you only use the` <p> ` tag for a paragraph it for a sematic reasons:+1:  Here above you can just use the simple `<div> or <h4>`

File: index.html [hunk 15]
```diff
@@ -39,26 +39,24 @@ <h1>I'm Tshepo <br> Glad to see <br> you!</h1>
...
-        <div class="Project-Text">
-            <h1>Tonic</h1>
+    <div class="Project-Text">
+            <h2>Tonic</h2>
 
     <ul class="Pro-P">
-        <li> <img class="Canopy" src="image/Text 4.png" alt="CANOPY"/> </li>
-
-        <li> <img class="counter-1" src="image/Counter2.png" alt="Counter"/> </li>
+        <li> <p class="Canopy">Canopy</p></li>
```
Comment (MaganaAsiati) [inline index.html:52]: - [ ] [ OPTIONAL] KindlyI suggest you only use the` <p> ` tag for a paragraph it for a sematic reasons:+1:  Here above you can just use the simple `<div> or <h4>`

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @jossi30, I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Kindly I suggest you make sure that the webpage takes the max screen in  the mobile phone at 100%  without the scrolling button from right to left or top to bottom . You still have that that you need to fix  :+1: 

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Kindly I suggest you use the appropriate font family It will make your design stick to the Figma design. I suggest you use the one used in the Figma design :+1: 

- [ ]  [OPTIONAL]  Kindly  I suggest you not use the picture for this section I shows you  below  just create the list of language and  add the border for each item and style it as it shows in the Figma design :+1: 

image

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] [OPTIONAL]  kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by jerryowusu — APPROVED** [review]
Hi @Tshepo45  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


CookedDelectableDolphin-size_restricted

## Highlights
All linters are passing :heavy_check_mark: 
Great effort so far  :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**